src/ansys/dpf/core/_version.py

```diff
@@ -14,6 +14,7 @@
     "5.0": ">=0.6.0",
     "6.0": ">=0.7.0",
     "7.0": ">=0.8dev",
+    "7.1": ">=0.8.1",
```

Comment:
@anslpa I thought this was not necessary anymore?

Reply:
@PProfizi internal CI/CD mechanism failed when I run it bumping the server version. Let me double check.

Reply:
Good catch, the fix was not that one.